util/app-config/src/cli.rs

```diff
@@ -266,3 +276,16 @@ fn init() -> App<'static, 'static> {
                 .hidden(true),
         )
 }
+
+fn is_hex(hex: &str) -> Result<(), String> {
```

Comment:
👍

Reply:
need to check hex char range, e.g. `is_hex("0xZZZ")` should return error.